### ISSUE TITLE
[Bug] Child entities bypass heat dissipation limits [20w22b]

### ISSUE THREAD

**Issue by Dwarf-LordPangolin** [tsunamayo/Starship-EVO#2224]
### Short Description of the issue:
Low-mass child entities can be used to bypass the limits of the heat mechanic.

### Steps to reproduce the issue:
This was discovered by fellow_starmadian on the Discord.

These two platforms are almost identical; one of them has sixteen 4 meter-long bricks on it, and the other has sixteen Small Rotators, with 3.75 meter-long bricks on it.

![20200526221304_1](https://user-images.githubusercontent.com/33041786/82973845-15f13580-9f9e-11ea-9f6c-9e19b21f2171.jpg)

The one without child entities can dissipate 435 heat per second:
![20200526221341_1](https://user-images.githubusercontent.com/33041786/82973881-2ef9e680-9f9e-11ea-807b-62c573d9e105.jpg)

The one with child entities can dissipate 2985 heat per second:
![20200526221347_1](https://user-images.githubusercontent.com/33041786/82973951-5650b380-9f9e-11ea-8b3c-45a1074a11e5.jpg)

This is because each child entity calculates its heat dissipation independently of its neighbors. Obviously, this can be scaled up to a much larger degree to allow players to place almost as many weapons, shields, or engines on their ships as they want. This neutralizes the usefulness of the heat system as a game mechanic.

### Environment
- Starship EVO version: 20w22b
- Graphics Card: NVIDIA GeForce GTX 1060 6GB

### Additional information:
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/4686281/output_log.txt)

**Comment by tsunamayo** [comment]
Yes I would have to do a single surface find on all entity at once - it is on my to-do list.
